sdk/keyvault/azure-security-keyvault-administration/swagger/autorest.md

```diff
@@ -1,79 +1,126 @@
 # Azure Key Vault Administration for Java
+
 > see https://aka.ms/autorest
 
-### Setup
-Increase max memory if you're using Autorest older than 3. Set the environment variable `NODE_OPTIONS` to `--max-old-space-size=8192`.
+This is the Autorest configuration file for KeyVault Administration.
 
-This is the AutoRest configuration file for the KeyVaultAccessControlClient and KeyVaultBackupClient.
 ---
 ## Getting Started
-To build the SDK for either client, simply [Install AutoRest](https://github.com/Azure/autorest/blob/master/docs/install/readme.md) and in this folder, run:
+To build the SDK for KeyVault Administration, simply [Install Autorest](https://aka.ms/autorest) and 
+in this folder, run:
 
-> `autorest`
+> `autorest --tag={swagger specification}`
 
 To see additional help and options, run:
 
 > `autorest --help`
 
-### Generation
-There are three swagger specifications for KeyVault Administration: `rbac`, `backuprestore` and `settings`. They use the following tags respectively: `--tag=rbac`, `--tag=backuprestore`, `--tag=settings`.
-
+### Setup
 ```ps
-cd <swagger-folder>
-autorest --use=@autorest/java@4.1.15 --tag=${package}
+npm install -g autorest
 ```
 
-e.g.
+### Generation
+
+There are three swagger specifications for KeyVault Administration: `rbac`, `backuprestore` and `settings`. 
+They use the following tags respectively: `--tag=rbac`, `--tag=backuprestore`, `--tag=settings`.
+
 ```ps
 cd <swagger-folder>
-autorest --use=@autorest/java@4.1.15 --tag=rbac
+autorest
 ```
 
+### Generation
+There are three swagger specifications for KeyVault Administration: 
+
 ```ps
 cd <swagger-folder>
-autorest --use=@autorest/java@4.1.15 --tag=backuprestore
+autorest --tag={swagger specification}
 ```
 
+e.g.
 ```ps
 cd <swagger-folder>
-autorest --use=@autorest/java@4.1.15 --tag=settings
+autorest --tag=rbac
+autorest --tag=backuprestore
+autorest --tag=settings
 ```
 
-## Code generation settings
-``` yaml
-java: true
+## Configuration
+```yaml
+use: '@autorest/java@4.1.16'
```

Comment:
Do we want to hard-code a version for these tags?

Reply:
I think so, as this is the same as an external dependency where we should explicitly be updating this when necessary to prevent random implicit changes.